### PR TITLE
Update the project.json metadata

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Fixes https://github.com/aspnet/dotnet-watch/issues/104

@Eilon please let me know if this is what you had in mind of you want me to rephrase or change the tags.

### CODE REVIEW

File: src/Microsoft.DotNet.Watcher.Tools/project.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "version": "1.0.0-*",
+  "description": "Dotnet watch CLI command.",
```
Comment (Eilon) [inline src/Microsoft.DotNet.Watcher.Tools/project.json]: Take some inspiration from this one: https://github.com/aspnet/Caching/blob/dev/src/Microsoft.Extensions.Caching.SqlConfig.Tools/project.json#L9

File: src/Microsoft.DotNet.Watcher.Core/project.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
 {
   "version": "1.0.0-*",
+  "description": "Dotnet watch core features. Contains the core logic for dotnet watch.",
```
Comment (Eilon) [inline src/Microsoft.DotNet.Watcher.Core/project.json]: How about: `Core logic for the dotnet-watch command line tool.` or something like that?

**Comment by Eilon** [comment]
🕐

**Comment by victorhurdugaci** [comment]
Updated

File: src/Microsoft.DotNet.Watcher.Tools/project.json [hunk 3]
```diff
@@ -1,5 +1,12 @@
 {
   "version": "1.0.0-*",
+  "description": "Command line tool to watch for source files changes during development and restart dotnet.",
```
Comment (Eilon) [inline src/Microsoft.DotNet.Watcher.Tools/project.json]: How about:
* `... source file changes...` (singular `file`).
 *`... and restart the dotnet command.`

**Comment by Eilon** [comment]
Small suggestion, then :shipit:

**Pull request merged**